src/test/java/HmWrk16.java

```diff
@@ -0,0 +1,48 @@
+import net.bytebuddy.asm.Advice;
+import org.checkerframework.checker.units.qual.K;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class HmWrk16 {
+
+    @Test
+    public static void createNewPlaylist() {
+
+        By usernameLocator = By.xpath("//*[@type='email']");
+        By passwordLocator = By.xpath("//*[@type='password']");
+        By loginButton = By.xpath("//*[@type='submit']");
+        By avatarIconLocator = By.xpath("//*[@alt = 'Avatar of Student']");
+        By NewPlaylistBtn = By.xpath("//i[@title='Create a new playlist']");
+        By createNewPlaylist = By.xpath("//*[contains(text(),'New Playlist')]");
+        By newPlaylistName = By.xpath("//*[@name='name']");
+        By playlistName = By.xpath("//*[contains(text(),'MySongs')]");
+
+        WebDriver driver = new ChromeDriver();
+        try {
+            driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+            String url = "https://bbb.testpro.io/";
+            driver.get(url);
+
+
+            driver.findElement(usernameLocator).sendKeys("demo@class.com");
+            driver.findElement(passwordLocator).sendKeys("te$t$tudent");
+            driver.findElement(loginButton).click();
+            driver.findElement(NewPlaylistBtn).click();
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/200116848-82ae585c-3c13-4622-8a23-b0436b980959.png)

_if your problem is still not solved, you need a DOM-level solution. So, we need an interface that helps to execute JavaScript with Selenium WebDriver. It is called JavaScriptExecutor. JavaScript works at the DOM level. Therefore, we are able to click any web element by using JavaScript executor._

_For, example_

  ```
((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView(true);", driver.findElement(NewPlaylistBtn));
   Thread.sleep(2000);
   ((JavascriptExecutor) driver).executeScript("arguments[0].click();", driver.findElement(NewPlaylistBtn));
```
![image](https://user-images.githubusercontent.com/104761482/200116916-dc0ed0f8-61ee-4751-84e8-87b4fc0c3b32.png)

How to use JavaScript in Selenium to click an Element?  [https://www.baeldung.com/java-selenium-javascript](url)